client/src/util/index.js

```diff
@@ -53,7 +53,7 @@ export const connectEthAccount = async () => {
 export const connectTezAccount = async () => {
   const client = new DAppClient({ name: "TEZEX" });
   const network =
-    config.tezos.conseilServer.network === "mainnet" ? "mainnet" : "florencenet";
+    config.tezos.conseilServer.network === "mainnet" ? "mainnet" : "granadanet";
```

Comment:
this should be a more generic "testnet" so it doesn't need to be changed every time a new testnet is deployed

Reply:
This is wallet specific, the beacon doesn't understand testnet it needs the network name

Reply:
in the config file there should be a `testnet` parameter with a 'granadanet' value